common/pom.xml

```diff
@@ -144,6 +149,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.xmlunit</groupId>
+            <artifactId>xmlunit-matchers</artifactId>
```

Comment:
Could you accomplish the fix without adding a new dependency to the project?

Reply:
I have fixed the issue without using the dependency. Pls review